mteb/leaderboard/figures.py

```diff
@@ -97,3 +97,90 @@ def performance_size_plot(df: pd.DataFrame) -> go.Figure:
         margin=dict(b=20, t=10, l=20, r=10),  # noqa
     )
     return fig
+
+
+TOP_N = 5
```

Comment:
May be moved lower and make it depend on something we'll change if we were to increase top N?
```suggestion
TOP_N = len(line_colors)
```